libs/wingsdk/src/cloud/secret.ts

```diff
@@ -79,15 +87,15 @@ export interface ISecretClient {
    * @Returns the secret value as string.
    * @inflight
    */
-  value(options?: GetSecretValueOptions): Promise<string>;
+  getValue(options?: ISecretGetValueOptions): Promise<string>;
```

Comment:
Why did we rename this?

I think this should still be `value()`

Reply:
Looking at all the other resources, we use verbs for defining the methods, so it made logically more sense to me. Having only `value` alone is something I would expect from a class attribute.